packages/realm-server/tests/server-endpoints-test.ts

```diff
@@ -321,98 +321,6 @@ module(basename(__filename), function () {
           }
         });
 
-        test('POST /_create-realm without copying seed realm', async function (assert) {
```

Comment:
this test is superfluous. it should have been removed when we removed seed realms